src/main/java/com/example/school/model/Grade.java

```diff
@@ -0,0 +1,8 @@
+package com.example.school.model;
+
+import java.util.List;
+
+public class Grade {
+    String name;
+    List<Student> students;
+}
```

Comment:
Let's remove the class for now and concentrate on classes `Student`,  `StudentDetails`.